gatsby/src/docs/workflow/integrations/integration-platform/ui-components/index.mdx

```diff
@@ -0,0 +1,296 @@
+---
+title: UI Components
+---
+
+The Sentry Integration Platform provides developers with the ability to define User Interface components they want to render within specific parts of Sentry's UI, through a JSON-based schema. Two possible use-cases are [Issue Link](#issue-link) and [Stack Trace Link](#stack-trace-link).
+
+# Schema
+
+The UI components are specified in the schema section of integration details. The required form of this schema is:
+
+```json
+{
+    "elements": [
+        // Each element is a specification for a component the developer wants to use.
+        ...
+    ]
+}
+```
+
+# Component Details
+
+## **Issue Link**
+
+With issue linking, the developer can connect Sentry issues with a task in their project management system. This functionality provides a way to use any project management tool the developer uses or develops.
+
+The Issue Link component displays "Link &lt;Service&gt; Issue" in the Issue sidebar, which opens a modal allowing the user to create or link the issue to a task in the external service. For example, this is what our Clubhouse integration looks like.
+
+![Sentry modal that creates a Clubhouse Story that's linked to a Sentry issue.](../../clubhouse/clubhouse_create_story_with_issue.png)
+
+**Schema**
+
+```json
+{
+    "type": "issue-link",
+    "link": {
+        "uri": <String>,
+        "required_fields": <Array<FormField>>,
+    "optional_fields": <Array<FormField>>
+    },
+    "create": {
+        // Same as "link" schema
+    }
+}
+```
+
+**Attributes of the link and create fields:**
+
+- uri - (Required) The URI to request when the User submits the Link/Create Issue form.
+- required_fields - (Required) List of `FormField` components the User is required to complete.
+- optional_fields - List of optional `Form Field` components the User *may* complete.
+
+A `FormField` can be one of three types (which behave like their HTML equivalents) — `Select`, `Text`, and `TextArea,` described below.
+
+## **Select**
+
+```json
```

Comment:
This should be ```js

Reply:
I believe these are all JSON Schema, but either way JSON is a subset of JavaScript, and these are just describing typed objects, no?

Reply:
I'm afraid I can't answer your reasonable question authoritatively. In practice, I have found that three backticks followed by "js" seems to render things properly. Using another form (jsx, for example) doesn't work.

Reply:
jsx might be invalid (unclear), but i can verify these. json schema is just json, but its possible because we're doing annotations here JS renders something that appears nicer. I'm fine optimizing to JS if that indeed ends up true.

Reply:
cool. I'll be curious to know if we're constrained to specifically using "js" - if so, we will capture that in our docs writing guide.